esi_leap/resource_objects/ironic_node.py

```diff
@@ -49,14 +49,12 @@ def get_contract_uuid(self):
 
     def get_project_id(self):
         node = get_ironic_client().node.get(self._uuid)
-        return node.properties.get('project_id', None)
+        return node.lessee
 
     def get_node_config(self):
         node = get_ironic_client().node.get(self._uuid)
         config = node.properties
         config.pop('contract_uuid', None)
-        config.pop('project_id', None)
-        config.pop('project_owner_id', None)
```

Comment:
Why get rid of these lines?

Reply:
Ah, this was the old way for an Ironic node to know who owned/leased it - a hackish way not supported upstream. This is now the official Ironic way to handle this.